docs/serverless/AI-for-security/ai-assistant.mdx

```diff
@@ -124,6 +127,10 @@ The **Settings** menu has the following tabs:
 
 ### Anonymization
 
+<DocCallOut title="Required permissions">
+To modify Anonymization settings, you need the **Elastic AI Assistant: All** privilege, with **Customize sub-feature privileges** enabled.
+</DocCallOut>
```

Comment:
I don't think these kinds of feature privileges are available in serverless yet. Currently privs are controlled solely through [prebuilt user roles](https://www.elastic.co/docs/current/serverless/general/assign-user-roles). (This is likely to change with future enhancements to serverless RBAC but we're not there yet.)